sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -154,24 +174,175 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
             .select("COUNT(*)")
             .from(EducationProgramOffering, "offerings")
             .where("offerings.educationProgram.id = programs.id")
+
+            .andWhere("offerings.offeringType in (:...offeringTypes)", {
+              offeringTypes,
+            })
             .andWhere("offerings.institutionLocation.id = :locationId", {
               locationId,
             }),
         "totalofferings",
       )
-      .where("programs.institution.id = :institutionId", { institutionId })
-      .getRawMany();
+      .where("programs.institution.id = :institutionId", { institutionId });
+
+    const queryParams: any[] = [...offeringTypes, locationId, institutionId];
+    // program name search
+    if (paginationOptions.searchName) {
+      summaryResult.andWhere("programs.name Ilike :searchProgramName", {
+        searchProgramName: `%${paginationOptions.searchName}%`,
```

Comment:
I believe that if the :searchProgramName is going t be provided as a global parameter in queryParams there is no need to provide it at line 192.